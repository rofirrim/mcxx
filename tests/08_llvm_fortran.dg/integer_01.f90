! <testinfo>
! test_generator="config/mercurium-fortran-llvm run"
! test_stdout="integer_01.f90.out"
! </testinfo>
PROGRAM MAIN
      IMPLICIT NONE
      INTEGER :: A
      INTEGER :: B
      INTEGER :: C

      A = 2
      PRINT *, A
      B = 5
      PRINT *, B

      C = A + B
      PRINT *, C

      C = B - A
      PRINT *, C

      C = A * B
      PRINT *, C

      C = B * A
      PRINT *, C

      C = B / A
      PRINT *, C

      C = A / B
      PRINT *, C
    
      C = -B
      PRINT *, C

      C = -B / A
      PRINT *, C

      C = B / (-A)
      PRINT *, C
END PROGRAM MAIN
