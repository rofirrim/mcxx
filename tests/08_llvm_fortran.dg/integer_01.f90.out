           2
           5
           7
           3
          10
          10
           2
           0
          -5
          -2
          -2
